ecl/hthor/hthor.cpp

```diff
@@ -7738,6 +7738,11 @@ void CHThorWSCBaseActivity::init()
         JBASE64_Encode(uidpair.str(), uidpair.length(), authToken, false);
     }
     soapTraceLevel = agent.queryWorkUnit()->getDebugValueInt("soapTraceLevel", 1);
+    StringBuffer soapSepStr;
+    StringBufferAdaptor soapSepAdaptor(soapSepStr);
+    agent.queryWorkUnit()->getDebugValue("soapLogSepString", soapSepAdaptor);
+    if (!soapSepStr.isEmpty())
+        soapSepString.clear().append(soapSepStr);
```

Comment:
calling here (in activity init code) means it this will be called multiple times per job (if there are act based on this base per job). [ that's true of existing soapTraceLevel line too ]

Maybe nothing to worry about